cli/cmd/plugin/ui/web/tanzu-ui/src/state-management/reducers/Form.reducer.tsx

```diff
@@ -15,14 +15,14 @@ interface FormState {
 export function formReducer(state: FormState, action: Action) {
     let newState;
     switch (action.type) {
-    case INPUT_CHANGE:
-        newState = {
-            ...state,
-            [action.field]: action.payload,
-        };
-        break;
-    default:
-        newState = { ...state };
+        case INPUT_CHANGE:
```

Comment:
I thought the (relatively new) standard was to have the `case` statements aligned with the `switch` statement